scripts/doc-generation/.jazzy.yaml

```diff
@@ -0,0 +1,11 @@
+module: MapboxMaps
```

Comment:
yamllint requests that the first line of this file be `---`

Reply:
Didn't know it! Fixed